src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py

```diff
@@ -40,9 +42,15 @@ def acr_run(cmd,  # pylint: disable=too-many-locals
     _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name, RUN_NOT_SUPPORTED)
 
+    platform_os, platform_arch, platform_variant = get_validate_platform(cmd, os_type, platform)
+
+    EncodedTaskRunRequest, FileTaskRunRequest, PlatformProperties = cmd.get_models('EncodedTaskRunRequest', 'FileTaskRunRequest', 'PlatformProperties')
+
     client_registries = cf_acr_registries(cmd.cli_ctx)
 
-    if os.path.exists(source_location):
+    if source_location == NULL_SOURCE_LOCATION:
```

Comment:
Is the string comparison case-sensitive?

Reply:
The string comparison is case-sensitive.